index.html

```diff
@@ -34,6 +34,155 @@ <h1>Charlie<br>Giorgania</h1>
+      <div class="code-buttons">
+        <ul>
+          <li><button>HTML/CSS</button></li>
+          <li><button>Ruby on Rails</button></li>
+          <li><button>Javascript</button></li>
+        </ul>  
+      </div>
+      <button class="arrow-click">See this project <img src="Images/Arrow.png" alt=""></button>
+    </div>
+  </div>
+  <div >
+    <figure class="wrapper"> 
+      <img src="Images/project-image-3.jpg" class="blockImage" alt="">
+    </figure>
+    <div class="grid-container">
+      <p>Project name goes here</p>
+      <div class="code-buttons">
+        <ul>
+          <li><button>HTML/CSS</button></li>
+          <li><button>Ruby on Rails</button></li>
+          <li><button>Javascript</button></li>
+        </ul>  
+      </div>
+      <button class="arrow-click">See this project <img src="Images/Arrow.png" alt=""></button>
+    </div>
+  </div>
+  <div >
+    <figure class="wrapper"> 
+      <img src="Images/project-image-4.jpg" class="blockImage" alt="">
+    </figure>
+    <div class="grid-container">
+      <p>Project name goes here</p>
+      <div class="code-buttons">
+        <ul>
+          <li><button>HTML/CSS</button></li>
+          <li><button>Ruby on Rails</button></li>
+          <li><button>Javascript</button></li>
+        </ul>  
+      </div>
+      <button class="arrow-click">See this project <img src="Images/Arrow.png" alt=""></button>
+    </div>
+  </div>
+  <div >
+    <figure class="wrapper"> 
+      <img src="Images/project-image-5.jpg" class="blockImage" alt="">
+    </figure>
+    <div class="grid-container">
+      <p>Project name goes here</p>
+      <div class="code-buttons">
+        <ul>
+          <li><button>HTML/CSS</button></li>
+          <li><button>Ruby on Rails</button></li>
+          <li><button>Javascript</button></li>
+        </ul>  
+      </div>
+      <button class="arrow-click">See this project <img src="Images/Arrow.png" alt=""></button>
+    </div>
+  </div>
+  <div>
+    <figure class="wrapper"> 
+      <img src="Images/project-image-6.jpg" class="blockImage" alt="">
+    </figure>
+    <div class="grid-container">
+      <p>Project name goes here</p>
+      <div class="code-buttons">
+        <ul>
+          <li><button>HTML/CSS</button></li>
+          <li><button>Ruby on Rails</button></li>
+          <li><button>Javascript</button></li>
+        </ul>  
+      </div>
+      <button class="arrow-click">See this project <img src="Images/Arrow.png" alt=""></button>
+    </div>
+  </div>
+</section>
+<section class="content">
+  <header>
+    <h1>About</h1>   
+  </header>
+  <p>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+    sed do eiusmod tempor incididunt ut labore et dolore magna 
+    aliqua. Ut enim ad minim veniam, quis nostrud exercitation 
+    ullamco laboris nisi ut aliquip ea eiusmod
+  </p> 
+  <button class="downloadcvbutton" type="button"><span class="collaboration">Get my CV</span><img src="svg/ic_download.svg" alt=""></button>
```

Comment:
- [ ] For this button, add the required button styling based on the design: Active, Hover and Pressed